requirements/requirements-dev.txt

```diff
@@ -147,7 +147,7 @@ ruff==0.7.2
     # via idmodels (pyproject.toml)
 s3fs==2024.10.0
     # via iddata
-sarix @ git+https://github.com/elray1/sarix@1c8995942d49afbb66637f0dc0662e1248606af4
+sarix @ git+https://github.com/elray1/sarix@c8921734a4328a372cf23ff343e781bc08dd4d36
```

Comment:
small suggestion (not part of a larger review yet) but I think we are moving towards hosting our own version of sarix here: https://github.com/reichlab/sarix/tree/main and not pointing at Evan's repo anymore...